src/styles/blocks/star.css

```
@@ -0,0 +1,26 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+
+  background-image: url(../../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:nth-child(-n+4) {
+  background-image: url("../../images/star-active.svg");
+}
+
+.rate__reviews {
+  color: var(--color-dark-blue);
+  line-height: var(--line-small);
+  font-weight: var(--font-weight-small);
+  font-size: var(--font-small);
+  margin: 0;
+  padding: 0;
+}
```

Same here. By naming it `rate__reviews` you need to have those in `rate`  block styles file.
One block - one file.
```suggestion
.rate__reviews {
  color: var(--color-dark-blue);
  line-height: var(--line-small);
  font-weight: var(--font-weight-small);
  font-size: var(--font-small);
  margin: 0;
  padding: 0;
}
```